hugo/content/exporter/_index.md

```diff
@@ -35,16 +35,18 @@ If you install the below available packages with RPM, you can continue reading a
 | pgmonitor-pg-common            | Package containing postgres_exporter items common for all versions of postgres |
 | pgmonitor-node_exporter-extras | Crunchy optimized configurations for node_exporter                        |
 | pg_bloat_check                 | Package for pg_bloat_check script                                         |
+| pgbouncer_fdw                  | Package for the pgbouncer_fdw extension                                   |
```

Comment:
do we have rpm ? Please RPM dependency

Reply:
No RPMs yet. Was waiting for PR review.

Reply:
pgb fdw is already released, correct ?

Reply:
No, it is not. I've been waiting for someone to test an installation of pgbouncer_fdw with pgmonitor to ensure there's nothing else that needs to to be fix with the fdw before release.